release/license.tpl

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Bazel Authors. All rights reserved.
+# Copyright 2019 The Bazel Authors. All rights reserved.
```

Comment:
Don't change the license year here. It should be the time when Bazel is first launched.

Reply:
We generally follow the rule of using the year when the product is first launched. It's sometimes not clear whether we should use 2016 or 2017 in this repo. I think we use 2016 for Bazel generated configs and 2017 for rules that we wrote and is specific to this repo (which is launched in 2017). For new repos to be created, we will use 2019.